### PR TITLE
Fix post install steps

### PR DESCRIPTION
**Pull request by rosieyohannan**
Backup and restore was missing from the post install phase of the installation guide.

Also - changes build environment to execution environment for consistency.

### CODE REVIEW

File: jekyll/_cci2/server-3-install-post.adoc [hunk 1]
```diff
@@ -16,6 +16,477 @@ toc::[]
 
 == Phase 4: Post installation
 
+=== Set up backup and restore
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-post.adoc:19]: These steps are lifted straight from the current backup and restore guide

File: jekyll/_cci2/server-3-install-prerequisites.adoc [hunk 2]
```diff
@@ -45,8 +45,28 @@ Download and install the following software before continuing:
 | KOTS: https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_darwin_amd64.tar.gz[Mac] or https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_linux_amd64.tar.gz[Linux]. 
 | {kotsversion}
 | Replicated Kubernetes Application Management
+
+| https://velero.io/docs/v1.6/basic-install/[Velero CLI] 
+| TBC
+| TBC
```
Comment (ryanwohara) [inline jekyll/_cci2/server-3-install-prerequisites.adoc:51]: The latest version is v1.7.1, which might mean this URL is already outdated.

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-prerequisites.adoc:51]: Maybe I could point here instead? https://github.com/vmware-tanzu/velero/releases

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-prerequisites.adoc:51]: @ryanwohara ^

Comment (ryanwohara) [reply jekyll/_cci2/server-3-install-prerequisites.adoc:51]: @rosieyohannan Sounds good to me! :)

File: jekyll/_cci2/server-3-install-prerequisites.adoc [hunk 3]
```diff
@@ -45,8 +45,28 @@ Download and install the following software before continuing:
 | KOTS: https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_darwin_amd64.tar.gz[Mac] or https://github.com/replicatedhq/kots/releases/download/v1.47.3/kots_linux_amd64.tar.gz[Linux]. 
 | {kotsversion}
 | Replicated Kubernetes Application Management
+
+| https://velero.io/docs/v1.6/basic-install/[Velero CLI]
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-prerequisites.adoc:49]:
```suggestion
| https://github.com/vmware-tanzu/velero/releases[Velero CLI] 
```

File: jekyll/_cci2/server-3-install-prerequisites.adoc (hunk 2 above)
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-prerequisites.adoc:51]:
```suggestion
| Latest
| Backup and restore capability
```

**Review by devinmarieb — APPROVED** [review]
My only feedback is to maybe change the section titles on lines 153 and 158 in the server-3-install.adoc file for consistency. (Save and Deploy / Create DNS Entry)

**Comment by rosieyohannan** [comment]
> My only feedback is to maybe change the section titles on lines 153 and 158 in the server-3-install.adoc file to sentence case for consistency. (Save and Deploy / Create DNS Entry)

We have a lot of inconsistency around this. Tyler wrote a script to sort them all out so I'll ask him where it is...

File: jekyll/_cci2/server-3-install.adoc [hunk 4]
```diff
@@ -150,12 +150,12 @@ Number of days to retain your test results and artifacts. Set to 0 to disable an
 
 You can skip these sections unless you plan on using an existing Postgres, MongoDB or Vault instance, in which case see the https://circleci.com/docs/2.0/server-3-operator-externalizing-services/[Externalizing Services doc]. By default CirecleCI server will create its own Postgres, MongoDB and Vault instances within the CircleCI namespace. The instances inside the CircleCI namespace will be included in the CircleCI backup and restore process. 
 
-==== Save and Deploy
+=== Save and Deploy
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:153]:
```suggestion
=== Save and deploy
```

File: jekyll/_cci2/server-3-install.adoc [hunk 5]
```diff
@@ -150,12 +150,12 @@ Number of days to retain your test results and artifacts. Set to 0 to disable an
...
 Once you have completed the fields detailed above it is time to deploy. The deployment will install the core services and provide you an IP address for the Traefik load balancer. That IP address will be critical in setting up a DNS record and completing the first phase of the installation. 
 
 NOTE: In this first stage we skipped a lot of fields in the config. Not to worry. We will revisit those in the next stages of installation.
 
-==== Create DNS Entry 
+=== Create DNS Entry
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install.adoc:158]:
```suggestion
=== Create DNS entry 
```

**Pull request merged**